script/Ciel/Day 8/08_01_CIEL08_3B.txt

```diff
@@ -317,47 +342,52 @@ It was a lightning-fast movement, almost as if it was of the utmost importance.
 [sha:23546edcd305ad726c2e848c1b841aadd0501133]{
 -- Page 273, Offset 18950.
 -- うん、ホントに大漁でした。
-Yeah, I've seriously hauled in a good catch.
+Yeah, I've landed a big catch alright.
```

Comment:
senpai is a big fish

Reply:
Is this a comment or a suggestion?

Reply:
the old version of the line. its my favorite mis-tl